superset/assets/javascripts/SqlLab/components/QueryAutoRefresh.jsx

```diff
@@ -16,11 +17,18 @@ class QueryAutoRefresh extends React.PureComponent {
   componentWillUnmount() {
     this.stopTimer();
   }
+  setPotentialTimeout(query) {
+    // if a running query is over 24 hrs ago, set the state timeout
+    if (query.state === 'running' && addXHours(query.startDttm, 24) < now()) {
```

Comment:
I would like for this to be aligned with the backend:
SQLLAB_ASYNC_TIME_LIMIT_SEC